packages/apollos-ui-connected/src/RockAuthedWebBrowser/index.js

```diff
@@ -28,17 +34,23 @@ const RockAuthedWebBrowserWithClient = ({ children, client, paper, primary }) =>
             forceCloseOnRedirection: false,
           },
         }),
+        ...Platform.select(browserOptions),
         ...iABOptions,
       },
       authOptions
     )
   );
 
 const RockAuthedWebBrowser = withApollo(
-  withTheme(({ theme: { colors: { paper, primary } = {} } = {} }) => ({
-    paper,
-    primary,
-  }))(RockAuthedWebBrowserWithClient)
+  withTheme(
+    ({
+      theme: { colors: { paper, primary } = {}, browserOptions = {} } = {},
+    }) => ({
+      paper,
+      primary,
+      browserOptions,
+    })
+  )(RockAuthedWebBrowserWithClient)
```

Comment:
```suggestion
    withTheme(
    ({ theme: { colors: themeColors = {} } = {} }) => ({
      themeColors,
    }),
    'ui-connected.RockAuthedWebBrowser'
  )(RockAuthedWebBrowserWithClient)
```

Line 51 above enables prop overrides in theme!